clients/fides-js/src/components/fides.css

```diff
@@ -2,62 +2,112 @@
 * Default CSS styles for the banner
 */
 :root {
-  /* Colors */
+
+  /* Base Colors */
```

Comment:
it might be worth adding a comment to say that the vars here are the ones to override